app/models/warehouse_report/outcomes/base.rb

```diff
@@ -308,8 +308,7 @@ def destinations
     end
   end
 
-  def destination_bucket(client_id, dest_id)
-    return 'returned to shelter' if returns_to_shelter_after_ph.keys.include?(client_id)
+  def destination_bucket(dest_id)
```

Comment:
With this removed from here, should _A return to homelessness is defined as any subsequent night in an Emergency Shelter or on the street, either directly after exiting, or after exiting to a permanent housing destination._ be removed from the Destination at Exit text?